integrationTests/multiShard/smartContract/executingSCCalls_test.go

```diff
@@ -44,7 +44,7 @@ func deploySmartContract(t *testing.T, nodeToProcess *testNode, roundNumber uint
 		vm.CreateEmptyAddress().Bytes(),
 		senderNonce,
 		big.NewInt(0),
-		scCode,
+		scCode+"@00",
```

Comment:
might replace the hardcoded 00 with procFactory.IELEVirtualMachine

Reply:
done